### PR TITLE
PCM-7736: fixed case description on draft case not saving

### PR DESCRIPTION
**Pull request by jeudy100**

### CODE REVIEW

**Comment by anujsi** [comment]
1 review comment. Please check

**Comment by jeudy100** [comment]
@anujsi I don't see a comment, can you repost it?

**Comment by anujsi** [comment]
@jeudy100 
You are getting a description here because you are using ".text" in the end.
Could you please check.
var draftNewCase = CaseService.localStorageCache.get(securityService.loginStatus.authedUser.sso_username).text;

File: app/cases/controllers/new.js [hunk 1]
```diff
@@ -305,11 +305,12 @@ export default class New {
         $scope.initDescription = function () {
             var searchObject = $location.search();
             if (RHAUtils.isNotEmpty(CaseService.localStorageCache) && CaseService.localStorageCache.get(securityService.loginStatus.authedUser.sso_username)) {
-                CaseService.kase.description = CaseService.localStorageCache.get(securityService.loginStatus.authedUser.sso_username).text;
-                CaseService.kase.problem = CaseService.localStorageCache.get(securityService.loginStatus.authedUser.sso_username).problem;
-                CaseService.kase.environment = CaseService.localStorageCache.get(securityService.loginStatus.authedUser.sso_username).environment;
-                CaseService.kase.occurance = CaseService.localStorageCache.get(securityService.loginStatus.authedUser.sso_username).occurance;
-                CaseService.kase.urgency = CaseService.localStorageCache.get(securityService.loginStatus.authedUser.sso_username).urgency;
+                var draftNewCase = CaseService.localStorageCache.get(securityService.loginStatus.authedUser.sso_username).text;
```
Comment (jeudy100) [inline app/cases/controllers/new.js]: @anujsi if you look at the function $scope.getLocalStorageForNewCase() there it access the localstorage the same way but it is done the new way I added. I'm fairly certain the other way was not correct.

**Comment by anujsi** [comment]
@jeudy100 Reviewed. Please test it properly and merge it.

**Pull request merged**